app/assets/stylesheets/themes/default/_ad.scss

```diff
@@ -4,12 +4,11 @@
   }
 }
 
-.promotion .ad {
-  text-align: center;
-  @include media($smaller-tablet) {
-    img {
-      width: 100%;
-      max-width: 350px;
+.promotion {
+  &--ad {
+    margin-top: 20px;
+    & > * {
+      margin: 0 auto;
```

Comment:
We can't use this technique if we use adsense because adsense's tag has `style="display: inline-block; width: 300px; height: 250px;"` :cry: 

I don't know the way to center inner element without `!important` hack except to use flexbox:

```css
    display: flex;
    justify-content: center;
```